packages/mcp/src/auth/router.ts

```diff
@@ -0,0 +1,192 @@
+import { TooManyRequestsError } from '@modelcontextprotocol/sdk/server/auth/errors.js'
+import type { OAuthServerProvider } from '@modelcontextprotocol/sdk/server/auth/provider.js'
+import { createOAuthMetadata } from '@modelcontextprotocol/sdk/server/auth/router.js'
```

Comment:
Can we avoid depending on Node.js-specific APIs?  The error has been thrown on Wrangler without enabling `nodejs_compat`.

<img width="2310" height="2896" alt="CleanShot 2025-10-31 at 16 39 50@2x" src="https://github.com/user-attachments/assets/fd110f2b-cb5d-4f03-8be9-7e9164843daf" />

For example, `@modelcontextprotocol/sdk/server/auth/router.js` depends on Express.